3-networks/modules/dedicated_interconnect/README.md

```diff
@@ -0,0 +1,25 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
We should provide a brief overview of what the module is designed to do, even if only a few sentences.

Reply:
Added an explanation and link to public documentation and also added commented example calls from the environments.